src/FiniteStrain/stvenant.jl

```diff
@@ -0,0 +1,38 @@
+"""
+    StVenant(; E, ν)
+
+Hyperelastic material
+#Arguments
+- `λ::Float64`: Lamé parameter
+- `μ::Float64`: Lamé parameter (shear modulus)
+"""
+
+struct StVenant <: AbstractMaterial
+    λ::Float64
+    μ::Float64
+end
+
+struct StVenantState <: AbstractMaterialState
+end
+
+function initial_material_state(::StVenant)
+    return StVenantState()
+end
+
+function StVenant(; λ::T, μ::T) where T
+    return StVenant(λ, μ)
+end
+
```

Comment:
What do you think about collecting the strain energy functions for all finite strain models? It would be more consistent but not only that, we could provide nice tests with AD later.

Reply:
I like the idea, but I could only find the strain energy in terms of Green-lagrange strain. Should be easy enough to derive it in terms of C, but I will merge it as it is now and someone can make a PR for this.